Actions/RunPipeline/RunPipeline.ps1

```diff
@@ -135,7 +135,7 @@ try {
             $buildAll = Get-BuildAllApps -baseFolder $baseFolder -project $project -modifiedFiles $modifiedFiles
         }
         catch {
-            OutputWarning -message "Failed to calculate modified files since $baselineWorkflowSHA, building all apps"
+            OutputNotice -message "Failed to calculate modified files since $baselineWorkflowSHA, building all apps"
```

Comment:
This should stay a warning. 

Maybe even an error?
At this point modified files should be known, so there should be no reason for failure (either an instability or a bug). So, one might want to re-run the build instead of building all apps.

Reply:
The primary reason for this situation is a. A rebase or b. Too many modified files. In both cases what you want is not an error or a warning, instead you want a full rebuild

Reply:
But this should all be determined during _Initialization_ job, not here.

I also don't see how `Get-ModifiedFiles` handles rebases or too many files. 🤔

Reply:
This PR doesn't change the logic, it just changes the logs.

Get-ModifiedFiles doesn't handle rebases or too many files, but...
If you are trying to do a git diff with a SHA, which isn't in the history, it will fail.
If you are trying to do a git diff with a changelist which exceeds memory usage, it will fail.
In these cases - a full build is better than an error, which the user doesn't have a clue what to do about.

Reply:
> a full build is better than an error, which the user doesn't have a clue what to do about.

I'm not sure about that. A full build due to an instability might have unwanted consequences. The user might prefer to simply re-run the build.

Anyway, let's take this discussion offline.